src/Illuminate/Database/Console/Migrations/ResetCommand.php

```diff
@@ -47,12 +47,12 @@ public function __construct(Migrator $migrator)
     /**
      * Execute the console command.
      *
-     * @return void
+     * @return int|void
```

Comment:
`int|void` is not a valid return type.

Reply:
From what point of view ?

I would agree to add a `return 0;` and then use `@return int`
Is that what you would want ?

Reply:
> From what point of view ?

`void` cannot appear in a union type in PHP. https://wiki.php.net/rfc/union_types_v2

> I would agree to add a return 0; and then use @return int

Yes, that's the only thing we can do here. Please do that.

Reply:
Done in f59885b544c7b4516d68ee38a68a507816a0febf

Thank you for the documentation link :)